docs/command-palette-guide.md

```
@@ -0,0 +1,462 @@
+  },
+  {
+    "prefix": "@altphone",
+    "label": "Alt Phone",
+    "searchType": "patientAttribute",
+    "fieldType": "person",
+    "fieldsToSearch": ["alternatePhoneNumber"]
+  }
+]
+```
+
+The user types `@altphone ` followed by the number to search by alternate phone.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)